data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered) VALUES ('Agumon', date '2020-02-03', 10.23, true);
```

Comment:
- [ ] Agumon is missing "escape attempts".  Please, add the value (should be explicitly zero). 

Example: 

```SQL
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

Reply:
Thanks a lot